packages/cmf/src/cmfConnect.js

```diff
@@ -245,7 +245,10 @@ export default function cmfConnect({
 					this.dispatchActionCreator(
 						'cmf.saga.start',
 						{ type: 'DID_MOUNT', componentId: this.id },
-						this.props,
+						{
+							componentId: this.props.componentId || 'default',
```

Comment:
can you reuse the function getComponentId in CmfConnect to get the componentId.

Reply:
How my custom saga can use the props ?

Reply:
I have updated the doc